modules/smart-agent_cassandra-nodetool/conf/00-status.yaml

```
@@ -0,0 +1,18 @@
+## Example
+module: cassandra nodetool
+name: "Node is not in live status"
+transformation: ".min(over='10m')"
+
+aggregation: false
```

I would set it to true to give the user ability to cusotmize this from variable if needed:

```suggestion
aggregation: true
```